cpp/doxygen/developer_guide/TESTING.md

```diff
@@ -79,6 +78,7 @@ the same tests across multiple types, we use GTest's
 Typed tests allow you to write a test once and run it across a list of types.
 
 For example:
+
 ```c++
```

Comment:
I'm confused about what cases are supposed to use a raw indent to indicate "code block" and what cases are supposed to use triple-backticks. The triple backticks are my preference. Does Doxygen assume the language to use for syntax highlighting?

Reply:
> Does Doxygen assume the language to use for syntax highlighting?

I don't think so.

The above change of replacing backticks with indent might be a leftover.

Reply:
https://github.com/rapidsai/cudf/pull/11475#discussion_r957902484

Reply:
In summary: I also prefer the backticks, but I used the indented code blocks in DOCUMENTATION.md so that doxygen wouldn't interpret doxygen commands inside the Markdown file.

Reply:
Thanks. That's fine.